ion-schema/src/main/kotlin/com/amazon/ionschema/internal/SchemaContentCache.kt

```diff
@@ -75,7 +76,7 @@ internal class SchemaContentCache constructor(
      * into this instance of [SchemaContentCache].
      */
     fun doesSchemaDeclareType(schemaId: String, typeId: IonSymbol): Boolean {
-        return runCatching { typeId in getSchemaContent(schemaId).declaredTypes }.getOrElse { false }
+        return runCatching { typeId.withoutTypeAnnotations() in getSchemaContent(schemaId).declaredTypes }.getOrElse { false }
```

Comment:
🗺️ Comparing the `typeId` symbol with the annotations intact was causing e.g. `nullable::foo` to not be found even though `foo` type existed.